susemanager-utils/testing/docker/README.md

```diff
@@ -6,57 +6,48 @@ used by our testing infrastructure.
 This is the hierarchy of our containers:
 
 ```
-                         +----------------+
-                         |  sle-<release> |
-                         +--------+-------+
-                                  |
-                                  |
-                                  |
-                       +----------+---------+
-                       | suma-<branch>-base |
-                       +----------+---------+
-                                  |
-          +-----------------------+----------------------+
-          |                       |                      |
-          |                       |                      |
-+---------+-----------+ +---------+---------+ +----------+----------+
-| suma-<branch>-pgsql | | suma-<branch>-ora | |suma-<branch>-nodejs |
-+---------------------+ +-------------------+ +---------------------+
+    +----------------+
+    |  opensuse/leap |
+    +--------+-------+
+             |
+             |
+             |
+  +----------+----------+
+  | uyuni-<branch>-base |
+  +----------+----------+
+             |
+             +-------------------+
+             |                   |
+             |                   |
++----------------------+ +-----------------------+
+| uyuni-<branch>-pgsql | | uyuni-<branch>-nodejs |
++----------------------+ +-----------------------+
 ```
 
-We have one SUMa specific base container per each branch (1.7, 2.1, ..., head).
-This container is based on "official" SLE container for the SUSE SLE release
-user by the tested branch. These containers are external to the SUMa project,
+We have one Uyuni specific base container per each branch (head).
+This container is based on "official" opensuse container for the release
+used by the tested branch. These containers are external to the Uyuni project,
 they are built with kiwi and we do not have to maintain them.
 
-All the SUMa-related containers are built using
+All the Uyuni-related containers are built using
 [docker's build feature](http://docs.docker.io/en/latest/use/builder/).
 
-From the `suma-<branch>-base` container we create three more containers:
+From the `suma-<branch>-base` container we create two more containers:
 
-  * `suma-<branch>-pgsql`
-  * `suma-<branch>-ora`
-  * `suma-<branch>-nodejs`
+  * `uyuni-<branch>-pgsql`
+  * `uyuni-<branch>-nodejs`
 
-These containers have all the packages required to test both the Java and the
-Python codebase against either PostgreSQL or Oracle.
+This containers has all the packages required to test both the Java and the
+Python codebase against PostgreSQL.
 
 ## Building
 
 Right now the base containers are build manually, just enter their directory and
 run:
 ```
-docker build -t suma-<branch>-<base|pgsql|ora|nodejs> .
```

Comment:
There are a couple of scripts that are related to oracle: docker-testing-oracle.sh, reset_oracle_database.sh,init-ora-db4eclipse.sh. Should these be removed as well ?

Reply:
Not now. This is for later when we are really sure that it is gone.